src/navigation/navigationContainer.tsx

```diff
@@ -117,6 +125,25 @@ export const NavigationWrapper = () => {
           options={{
             headerBackTitle: 'Back',
             headerTitle: 'Bird Watcher - Frigate',
+            headerLeft: () => {
+              const nav = useNavigation();
+
+              if (!config.data?.birdseye.enabled) {
+                return null;
+              }
+
+              return (
+                <BirdseyeIcon
+                  height={25}
+                  width={25}
+                  onPress={() => {
+                    nav.navigate('Birdseye');
+                  }}
+                  fill={tintColor}
+                  fillSecondary={tintColor}
+                />
+              );
+            },
             headerTintColor: isDarkMode
```

Comment:
Then we can use is like this in the component.
```suggestion
            headerLeft: props => HeaderLeftIcon(props),
```